storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/impl/DefaultStorIOSQLite.java

```diff
@@ -257,7 +257,16 @@ protected InternalImpl(@Nullable Map<Class<?>, SQLiteTypeMapping<?>> typesMappin
                 // Okay, we don't have direct type mapping.
                 // And we don't have cache for indirect type mapping.
                 // Let's find indirect type mapping and cache it!
-                Class<?> parentType = type.getSuperclass();
+
+                // Let's try to find indirect type mapping in own interfaces
+                for (Class<?> ownInterface : type.getInterfaces()) {
```

Comment:
1. Please add tests!
2. What about interfaces implemented by parent classes? I guess, we should inspect them too, though, we can start with this PR and add support for parent interfaces later if somebody will find a need in that